### DISCUSSION TITLE
Screenshot toggle

### DISCUSSION THREAD

**Discussion by Rswithel** [ente-io/ente#3139] [Enhancements]
Please make a feature in which you can toggle the feature for on and off for the screenshot in ente auth

--ente auth
--mobile

**Comment by schewara** [comment]
This would be also a highly wished feature from our end, to be able to provide nice screenshots for internal user documentation.

Does one know of any alternatives to get this done through some debug settings or developer options?

**Comment by vishnukvmd** [comment]
Once you disable Settings > Security > App lock, screenshots will work as expected.

**Comment by schewara** [comment]
@vishnukvmd Thank you that was the missing pointer I was missing and would be great to have also documented somewhere. :heart: 

@Rswithel seems the Settings > App lock > Hide content is the screenshot toggle you are looking for

There is just one part which does not make sense and is very confusing.

App lock was never activated or deactivated (the default on a fresh install), therefore Screenshots are not working by default.

It seems it needs an explicit activate + deactivate step to make screenshots actually work.
This inconsistent behavior for the disabled App lock is really confusing and should be addressed somehow.

While I highly honor the secure by default functionality some small mention in the Documentation of FAQ would go a long way here.

I am happy to open a PR for a new FAQ entry based on your feedback and thoughts.

**Reply by vishnukvmd** [reply]
Thank you, please feel free to create a pull request against this file: https://github.com/ente-io/ente/edit/main/docs/docs/auth/faq/index.md

**Reply by GeoQuack** [reply]
Thank you @schewara this worked for me. It would also be great to return to the default behaviour - no app lock, no screenshots, but there are times when the ability to make screenshots is helpful.
